codec_specs.md

```diff
@@ -220,6 +220,26 @@ Initialisation: The `Private Data` contains the fourcc as found in MP4 movies:
 
 [this page for more technical details on ProRes](http://wiki.multimedia.cx/index.php?title=Apple_ProRes#Frame_layout)
 
+### V_VP8
+
+Codec ID: V_VP8
+
+Codec Name: VP8 Codec format
+
+Description: VP8 is an open and royalty free video compression format developed by Google and created by On2 Technologies as a successor to VP7. [@!RFC6386]
+
+Initialisation: none
+
+### V_VP9
+
+Codec ID: V_VP9
+
+Codec Name: VP9 Codec format
+
+Description: VP9 is an open and royalty free video compression format developed by Google as a successor to VP8. [Draft VP9 Bitstream Overview](https://www.ietf.org/archive/id/draft-grange-vp9-bitstream-00.txt)
```

Comment:
I would rather use the more generic link http://www.webmproject.org/vp9/
It has the bitstream and how to put it in different containers. Also the draft is more recent.